src/dao/sqlUtils.ts

```diff
@@ -105,7 +92,7 @@ const decomposeWhere = (where: SqlWhere, table: string) => Object.keys(where ||
   }
 }, { conditions: [], values: [] })
 
-const parseResponse = (resultSet = []) => {
+const parseResponse = (resultSet: any[]) => {
   const resultMapped = resultSet.reduce((acc: any[], i: any) => {
     const newItem = { ...i }
```

Comment:
Doesn't 'i' needs to be an object?

Reply:
I put any because the object can be "a: { b: { } }" to avoid the type error I set as any